### PR TITLE
More PHP elements

### PR DESCRIPTION
**Pull request by JuGid**
- Functions
- Useful string manipulation (some are PHP >= 8.0)
- Match (php >= 8.0)
- Short syntax array
- Enums (php >= 8.1)

### CODE REVIEW

File: languages/php.php [hunk 1]
```diff
@@ -368,6 +459,48 @@ class ClassWithLogger
 }
 
 
+/**
+ * Enums (PHP >=8.1)
```
Comment (kang8) [inline languages/php.php:463]: Spaces typo: `PHP >= 8.1` to `PHP >= 8.1`

File: languages/php.php [hunk 2]
```diff
@@ -368,6 +459,48 @@ class ClassWithLogger
...
+ * https://www.php.net/manual/fr/language.types.enumerations.php
```
Comment (kang8) [inline languages/php.php:464]: English is better: https://www.php.net/manual/en/language.types.enumerations.php

File: languages/php.php [hunk 3]
```diff
@@ -97,6 +163,31 @@
 $argv; // Array of terminal arguments (filename included)
 $argc; // Number of arguments passed into terminal
 
+/**
+ * Functions
+ */
+
+ // Simple function
```
Comment (kang8) [inline languages/php.php:170]: Extra line first space.

**Review by kang8 — COMMENTED** [review]
some suggest

**Comment by JuGid** [comment]
Sorry for not answered this suggests @kang8 , gonna send a new pull request for this.

File: languages/php.php [hunk 4]
```diff
@@ -38,6 +65,25 @@
...
+if($i > 10) {
+
+} elseif( $i > 100) {
+
+} else {
+
+}
+
+// Ternary
+$string = $state == 'Running' ? 'He is running' : 'I don\'t know';
```
[the remark this answers is not in the document]
Comment (JuGid) [reply languages/php.php:82]: I will add something about the difference between == and ===, It seems important for beginners. Thanks !

**Comment by LeCoupa** [comment]
Thanks :)

**Pull request merged**